spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowScoreFunction.scala

```diff
@@ -61,10 +61,14 @@ class FlowScoreFunction(timeCuts: Array[Double],
 
     val zeroProb = Array.fill(topicCount) { 0.0 }
 
-    /** A null value for srcTopicMix or dstTopicMix indicated the ip (source or dest respectively)
+    /** word_error indicates there was a problem creating a word and should not be used for scoring.
+
+      A null value for srcTopicMix or dstTopicMix indicated the ip (source or dest respectively)
       * were not seen in training.
       */
-    if (srcTopicMix == null || dstTopicMix == null) {
+    if(srcWord == "word_error" || dstWord == "word_error"){
+      -1d
```

Comment:
if we are using -1 and 0 to encode "meanings" other than a floating point score, they should be named constants